src/pages/Home.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import '../Styles/Home.css'
+
+
+const Home = () => {
+    
+    
+    const [homeArticles, setHomeArticles] = useState([])
+
+
+    useEffect(() => {
+        axios.get('https://nc-news-fz7g.onrender.com/api/articles')
```

Comment:
Nice use of axios, our API calls should be separated out into its own file and put in functions which are exported, and imported where necessary
Arranging our code in this way means we would only need to import axios in one place